config/mcus/canboard_octopus.cfg

```diff
@@ -0,0 +1,56 @@
+[mcu]
+##--------------------------------------------------------------------
+# This board work by using a serial connection by default.
+# If you want to use CAN, invert the commented lines here
+
+# canbus_uuid: change-me-to-the-correct-canbus-id
+##--------------------------------------------------------------------
+
+
+[board_pins mcu]
+mcu: mcu
+aliases:
+
+# steppers
+   X_STEP=PF13, X_DIR=PF12, X_ENABLE=PF14, X_TMCUART=PC4,
+   Y_STEP=PG0, Y_DIR=PG1, Y_ENABLE=PF15, Y_TMCUART=PD11, Y_STOP=PG9,
+   Z_STEP=PF11, Z_DIR=PG3, Z_ENABLE=PG5, Z_TMCUART=PC6, Z_STOP=PG10,
+   Z1_STEP=PG4, Z1_DIR=PC1, Z1_ENABLE=PA0, Z1_TMCUART=PC7,
+   Z2_STEP=PF9, Z2_DIR=PF10, Z2_ENABLE=PG2, Z2_TMCUART=PF2,
+   Z3_STEP=PC13, Z3_DIR=PF0, Z3_ENABLE=PF1, Z3_TMCUART=PE4,
+
+# bed
+   BED_HEATER=PA1, BED_TEMPERATURE=PF3, 
+
+# sensor
+   CHAMBER_TEMPERATURE=PF6, ELECTRICAL_CABINET_TEMPERATURE=PF5,
+
+# recirculating filter
+   FILTER_FAN=P2.7,	
+	
+# Controller Fans
+   CONTROLLER_FAN1=PA8, CONTROLLER_FAN2=PE5,
+
+# EXHAUST FAN
+   EXHAUST_FAN=PD12,
+
+#lights
+   LIGHT_OUTPUT=PD15,
+   LIGHT_NEOPIXEL=PB6,
+
+# runout_sensor
+   RUNOUT_SENSOR=PG11,
+
+# EXP1 header
+   EXP1_1=PE8, EXP1_2=PE7,
+   EXP1_3=PE9, EXP1_4=PE10,
+   EXP1_5=PE12, EXP1_6=PE13,    # Slot in the socket on this side
+   EXP1_7=PE14, EXP1_8=PE15,
+   EXP1_9=<GND>, EXP1_10=<5V>,
+
+# EXP2 header
+   EXP2_1=PA6, EXP2_2=PA5,
+   EXP2_3=PB1, EXP2_4=PA4,
+   EXP2_5=PB2, EXP2_6=PA7,      # Slot in the socket on this side
+   EXP2_7=PC15, EXP2_8=<RST>,
+   EXP2_9=<GND>, EXP2_10=<5V>
```

Comment:
As stated in our last discussion: can you remove this file and just add the `canbus_uuid: change-me-to-the-correct-canbus-id` line to the original `octopus.cfg` file ?

Then I'll add some documentation on my side to the install note about it to say to add `toolhead:` for the pins of the extruder, hotend and everything that is plugged to the canboard if using one of the `canboard_xxx.cfg` file.

I think it's better to avoid duplicate to do it this way.

Reply:
Alright, is it okay for you if I add those pins as comments in the octopus file? (Reference)

Reply:
Yes I'm okay with that :)